src/lib.rs

```diff
@@ -16,6 +16,7 @@
 #![allow(unused_crate_dependencies)]
 #![warn(ffi_unwind_calls)]
 
+mod collector;
```

Comment:
Rename `collector` to `recorder`.
Rename `CollectorSender` to `Recorder`.
Rename `sender` to `recorder`.